CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing
+To contribute, submit a pull request.
+This project is under the AGPL v3+ license.
+If you do not agree to license your changes under this license, then do not make a pull request.
+Indeed, if you do not agree to the terms of this license, then you may not use the software.
+
+Include the following header for any new AGPL licensed files, and use your name as the author (update the year if you make a contribution between January 1st and us updating this document).
+If you modify an existing AGPL file, then add your name to the list of authors and update the year if applicable.
+
+```
+<one line to give the program's name and a brief idea of what it does.>
+Copyright (C) 2020  <name of author>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+```
+We also include the following [exception](https://www.gnu.org/licenses/gpl-faq.en.html#WMS) to the AGPL for this software.
+```
+As a special exception to the AGPL, any HTML file which merely makes function calls to this code, and for that purpose includes it by reference shall be deemed a separate work for copyright law purposes. In addition, the copyright holders of this code give you permission to combine this code with free software libraries that are released under the GNU LGPL. You may copy and distribute such a system following the terms of the GNU AGPL for this code and the LGPL for the libraries. If you modify this code, you may extend this exception to your version of the code, but you are not obligated to do so. If you do not wish to do so, delete this exception statement from your version.
+```
```

Comment:
Should the exception be placed in LICENSE?

Reply:
I thought we determined that we didn't need the exception actually.